packages/@aws-cdk/aws-sns/package.json

```diff
@@ -77,25 +77,22 @@
     "@aws-cdk/aws-cloudwatch": "^0.33.0",
     "@aws-cdk/aws-events": "^0.33.0",
     "@aws-cdk/aws-iam": "^0.33.0",
-    "@aws-cdk/aws-lambda": "^0.33.0",
-    "@aws-cdk/aws-sqs": "^0.33.0",
     "@aws-cdk/cdk": "^0.33.0"
   },
   "homepage": "https://github.com/awslabs/aws-cdk",
   "peerDependencies": {
     "@aws-cdk/aws-cloudwatch": "^0.33.0",
     "@aws-cdk/aws-events": "^0.33.0",
     "@aws-cdk/aws-iam": "^0.33.0",
-    "@aws-cdk/aws-lambda": "^0.33.0",
-    "@aws-cdk/aws-sqs": "^0.33.0",
     "@aws-cdk/cdk": "^0.33.0"
   },
   "engines": {
     "node": ">= 8.10.0"
   },
   "awslint": {
     "exclude": [
-      "construct-base-is-private:@aws-cdk/aws-sns.TopicBase"
+      "construct-base-is-private:@aws-cdk/aws-sns.TopicBase",
+      "integ-return-type:@aws-cdk/aws-sns.ITopicSubscription.bind"
```

Comment:
I think we should update the awslint rule to allow void as a return type for `bind`